test/runtests.jl

```diff
@@ -1,15 +1,10 @@
 using MLMetrics
 using MLMetrics.CompareMode
 
-if VERSION >= v"0.5-"
-    using Base.Test
-else
-    using BaseTestNext
-    const Test = BaseTestNext
-end
+using Test
 
 tests = [
-    "classification.jl",
```

Comment:
Commented out?

Reply:
Whoops, yeah, I did some testing and forgot to undo that.